thrifty-kotlin-codegen/src/test/kotlin/com/microsoft/thrifty/kgen/KotlinCodeGeneratorTest.kt

```diff
@@ -365,6 +365,62 @@ class KotlinCodeGeneratorTest {
         file shouldBe emptyList()
     }
 
+    @Test
+    fun `Emit @JvmName file-per-namespace annotations`() {
+        val thrift = """
+            |namespace kt test.consts
+            |
+            |const i32 FooNum = 42
+        """.trimMargin()
+
+        val text = generate(thrift) {
+                    emitJvmName()
+                    filePerNamespace()
+                }
+                .single()
+                .toString()
+
+        text shouldBe """
```

Comment:
These tests could be a little simpler with the `shouldStartWith` matcher (because we're only interested in the first line of output), but that doesn't seem to be available in this version of kotlintest (kotest).